### PR TITLE
Move `preload_stream` setting to entity settings

### PR DESCRIPTION
**Pull request by bramkragten**
## Breaking change

Entities without and unique id will lose this setting

## Proposed change

Move preload stream setting from more info dialog to entity settings.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/entities/entity-registry-settings.ts [hunk 1]
```diff
@@ -392,7 +419,23 @@ export class EntityRegistrySettings extends SubscribeMixin(LitElement) {
               @value-changed=${this._areaPicked}
             ></ha-area-picker>`
           : ""}
-
+        ${this._cameraPrefs
+          ? html`
+              <ha-settings-row>
+                <span slot="heading">Preload camera stream</span>
```
Comment (balloob) [inline src/panels/config/entities/entity-registry-settings.ts:425]: Localize.

File: src/panels/config/entities/entity-registry-settings.ts [hunk 2]
```diff
@@ -392,7 +419,23 @@ export class EntityRegistrySettings extends SubscribeMixin(LitElement) {
...
+                <span slot="description"
+                  >This keeps the camera stream open in the background so it
+                  shows quicker. Warning! This can slow down your instance
+                  drastically.</span
```
Comment (balloob) [inline src/panels/config/entities/entity-registry-settings.ts:429]: For in localize:
```suggestion
                  >This keeps the camera stream open in the background so it
                  shows quicker. Warning! This is CPU/network intensive.</span
```

**Review by balloob — DISMISSED** [review]

File: src/panels/config/entities/entity-registry-settings.ts (hunk 2 above)
Comment (zsarnett) [reply src/panels/config/entities/entity-registry-settings.ts:429]: We want to move from CPU though. Hardware /Network ?

Comment (matthiasdebaat) [reply src/panels/config/entities/entity-registry-settings.ts:429]: I agree on `Hardware`

Comment (balloob) [reply src/panels/config/entities/entity-registry-settings.ts:429]: I don't understand the comments. You're saying you want it to say "This is hardware intensive"?

Comment (matthiasdebaat) [reply src/panels/config/entities/entity-registry-settings.ts:429]: CPU is a bit technical in my opinion. Instead of hardware, maybe device?

**Review by zsarnett — APPROVED** [review]

**Pull request merged**